src/algebra/jordan/special.lean

```diff
@@ -0,0 +1,277 @@
+/-
```

Comment:
I think this file should wait for a follow-up PR. Then this PR can focus on getting `jordan.basic` right.

Reply:
This could almost be PR'd independently - only the last few lines actually use the new `jordan` structure

Reply:
I've created an independent PR for symmetrising and algebra #11399 and a third PR for introducing special Jordan algebras  #11401.